Classes/KIFUITestActor.m

```diff
@@ -1182,13 +1189,24 @@ - (void)scrollAccessibilityElement:(UIAccessibilityElement *)element inView:(UIV
 - (void)waitForFirstResponderWithAccessibilityLabel:(NSString *)label
 {
     [self runBlock:^KIFTestStepResult(NSError **error) {
-        UIResponder *firstResponder = [[[UIApplication sharedApplication] keyWindow] firstResponder];
-        if ([firstResponder isKindOfClass:NSClassFromString(@"UISearchBarTextField")]) {
-            do {
-                firstResponder = [(UIView *)firstResponder superview];
-            } while (firstResponder && ![firstResponder isKindOfClass:[UISearchBar class]]);
+        BOOL didMatch = NO;
+        UIResponder *foundResponder = nil;
+        for (UIResponder  *firstResponder in [[UIApplication sharedApplication] firstResponders]) {
+            foundResponder = firstResponder;
+
+            if ([foundResponder isKindOfClass:NSClassFromString(@"UISearchBarTextField")]) {
```

Comment:
:eyeroll: at this special case being here, but not for the other method w/ traits matching below... (no action for you, just noticing this while reviewing)

Reply:
Yeah I noticed a lot of inconsistencies around `accessibilityLabel` checking. Might warrant a deeper follow up to unify that logic.

Reply:
In general, `KIFUITestActor` is the old way of doing things and should eventually be deprecated. It probably isn't worth the time, as everything is already unified in `KIFUIViewTestActor` (new way).